plone/app/discussion/conversation.py

```diff
@@ -90,11 +90,19 @@ def enabled(self):
     @property
     def total_comments(self):
         public_comments = [
-            x for x in self._comments.values()
+            x for x in self.values()
             if user_nobody.has_permission('View', x)
         ]
         return len(public_comments)
 
+    def another_total_comments(self):
```

Comment:
@gforcada Why is there another_total_comments method? Why didn't you fix total_comments directly? Also it would be important to have a failing test to show what problem this pull request fixes.